### PR TITLE
Market

### PR DESCRIPTION
**Pull request by MrDenkoV**

### CODE REVIEW

File: frontend/views.py [hunk 1]
```diff
@@ -3,3 +3,7 @@
 
 def index(request):
     return render(request, 'frontend/index.html')
+
+
+def market(request):
+    return render(request, 'frontend/index.html')
```
Comment (maciektr) [inline frontend/views.py]: You should rather utilize the react router

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -1,16 +1,5 @@
...
-  python manage.py livereload
-else
-  python manage.py collectstatic --noinput
-  gunicorn enrollXchange.wsgi
-fi
-
+(cd frontend ; yarn install)
+(cd frontend ; yarn dev) &
+python manage.py runserver 0.0.0.0:8000 &
+python manage.py livereload
```
Comment (maciektr) [inline docker-entrypoint.sh]: You should not commit this changes. 
Please ignore it locally.

File: frontend/views.py (hunk 1 above)
Comment (MrDenkoV) [reply frontend/views.py]: Done.

File: docker-entrypoint.sh (hunk 2 above)
Comment (MrDenkoV) [reply docker-entrypoint.sh]: Cringe.

**Pull request merged**